<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1866</width>
    <height>1071</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Canvas" name="frame">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>90</y>
      <width>1461</width>
      <height>771</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="toolBoxFrame">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>151</width>
      <height>461</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>85</x>
       <y>0</y>
       <width>61</width>
       <height>411</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="freeHandButton">
        <property name="styleSheet">
         <string notr="true">QPushButton{
   background-color: rgb(85, 255, 255);
   color: #ffffff;
 
}

QPushButton:hover:!pressed {
    background-color: rgb(85, 170, 255);
    color: #ffffff;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="lineButton">
        <property name="styleSheet">
         <string notr="true">QPushButton{
   background-color: rgb(85, 255, 255);
   color: #ffffff;
 
}

QPushButton:hover:!pressed {
    background-color: rgb(85, 170, 255);
    color: #ffffff;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="circleButton">
        <property name="styleSheet">
         <string notr="true">QPushButton{
   background-color: rgb(85, 255, 255);
   color: #ffffff;
 
}

QPushButton:hover:!pressed {
    background-color: rgb(85, 170, 255);
    color: #ffffff;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="rectangleButton">
        <property name="styleSheet">
         <string notr="true">QPushButton{
   background-color: rgb(85, 255, 255);
   color: #ffffff;
 
}

QPushButton:hover:!pressed {
    background-color: rgb(85, 170, 255);
    color: #ffffff;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="dsFrame">
    <property name="geometry">
     <rect>
      <x>1700</x>
      <y>90</y>
      <width>151</width>
      <height>771</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QListWidget" name="dsList">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>151</width>
       <height>771</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="topFrame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>941</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="layoutWidget1">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>0</y>
       <width>371</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizonalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Color:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="penColorCB">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background: white;
color: black;</string>
        </property>
        <item>
         <property name="text">
          <string>Black</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Red</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Pink</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>White</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget2">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>0</y>
       <width>211</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_1">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Width: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="penWidthCB">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background: white;
color: black;</string>
        </property>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget3">
    <property name="geometry">
     <rect>
      <x>1700</x>
      <y>28</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="undoButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>21</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
    background-color: rgb(0, 255, 127);
    color: #ffffff;
    font-family: arial;
    font-weight: bold;
    text-decoration: none;
    outline: 0;
}

QPushButton:hover:!pressed {
    background-color: rgb(0, 255, 0);
    color: #ffffff;
}

QPushButton:disabled {
    background-color: rgb(153, 153, 153);
   color: #ffffff;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearSelected">
       <property name="styleSheet">
        <string notr="true">QPushButton{
    background-color: rgb(0, 255, 127);
    color: #ffffff;
    font-family: arial;
    font-weight: bold;
    text-decoration: none;
    outline: 0;
}

QPushButton:hover:!pressed {
    background-color: rgb(0, 255, 0);
   color: #ffffff;
}

QPushButton:disabled {
    background-color: rgb(153, 153, 153);
   color: #ffffff;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clear">
       <property name="styleSheet">
        <string notr="true">QPushButton{
    background-color: rgb(0, 255, 127);
    color: #ffffff;
    font-family: arial;
    font-weight: bold;
    text-decoration: none;
    outline: 0;
}

QPushButton:hover:!pressed {
    background-color: rgb(0, 255, 0);
   color: #ffffff;
}

QPushButton:disabled {
    background-color: rgb(153, 153, 153);
   color: #ffffff;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>layoutWidget3</zorder>
   <zorder>frame</zorder>
   <zorder>dsFrame</zorder>
   <zorder>topFrame</zorder>
   <zorder>toolBoxFrame</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1866</width>
     <height>18</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"> QMenuBar {
background-color: white;
color: black;
}

QMenuBar::item {
spacing: 3px; /* spacing between menu bar items */
padding: 1px 4px;
background: transparent;
border-radius: 4px;
}

QMenuBar::item:selected { /* when selected using mouse or keyboard */
background: #a8a8a8;
}

QMenuBar::item:pressed {
background: #888888;
}</string>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="styleSheet">
     <string notr="true">QMenu{
 background-color: rgb(238, 238, 238);
}

QMenu::item{
 background-color: rgb(238, 238, 238);
 color: black;
 }

 QMenu::item:selected:!disabled{
 background-color: rgb(0, 85, 127);
 color: rgb(255, 255, 255);
 } 

QMenu::item:disabled{
color: rgb(161, 161, 161);
}</string>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Ctril_J"/>
    <addaction name="actionSave_As_Ctrl_S"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="styleSheet">
     <string notr="true">QMenu{
 background-color: rgb(238, 238, 238);
}

QMenu::item{
 background-color: rgb(238, 238, 238);
 color: black;
 }

 QMenu::item:selected:!disabled{
 background-color: rgb(0, 85, 127);
 color: rgb(255, 255, 255);
 } 

QMenu::item:disabled{
color: rgb(161, 161, 161);
}</string>
    </property>
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy_Ctrl_C"/>
    <addaction name="actionCut_Ctrl_X"/>
    <addaction name="actionPaste_Ctrl_V"/>
    <addaction name="separator"/>
    <addaction name="actionUndo_Backspace"/>
    <addaction name="actionClear_selected_Ctrl_Backspace"/>
    <addaction name="separator"/>
    <addaction name="actionGroup_Ctrl_G"/>
    <addaction name="actionUngroup_Ctrl_U"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave">
   <property name="text">
    <string>Save As...  (Ctrl+S)</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open...  (Ctrl+J)</string>
   </property>
  </action>
  <action name="actionOpen_Ctril_J">
   <property name="text">
    <string>Open [Ctril+J]</string>
   </property>
  </action>
  <action name="actionSave_As_Ctrl_S">
   <property name="text">
    <string>Save As [Ctrl+S]</string>
   </property>
  </action>
  <action name="actionCopy_Ctrl_C">
   <property name="text">
    <string>Copy [Ctrl+C]</string>
   </property>
  </action>
  <action name="actionCut_Ctrl_X">
   <property name="text">
    <string>Cut [Ctrl+X]</string>
   </property>
  </action>
  <action name="actionPaste_Ctrl_V">
   <property name="text">
    <string>Paste [Ctrl+V]</string>
   </property>
  </action>
  <action name="actionUndo_Backspace">
   <property name="text">
    <string>Undo [Backspace]</string>
   </property>
  </action>
  <action name="actionClear_selected_Ctrl_Backspace">
   <property name="text">
    <string>Clear selected [Ctrl+Backspace]</string>
   </property>
  </action>
  <action name="actionGroup_Ctrl_G">
   <property name="text">
    <string>Group [Ctrl+G]</string>
   </property>
  </action>
  <action name="actionUngroup_Ctrl_U">
   <property name="text">
    <string>Ungroup [Ctrl+U]</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Canvas</class>
   <extends>QFrame</extends>
   <header>canvas.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
